src/main/java/org/kitteh/irc/client/library/command/ChannelModeCommand.java

```diff
@@ -122,12 +124,24 @@ public synchronized void execute() {
             this.getClient().sendRawLine("MODE " + this.getChannel());
             return;
         }
-        List<ModeStatus<ChannelMode>> queue = new ArrayList<>(MODES_PER_LINE);
+        int parameterModesPerLine = -1;
+        Optional<ISupportParameter.Modes> modes = this.getClient().getServerInfo().getISupportParameter("MODES", ISupportParameter.Modes.class);
```

Comment:
flatMap can't go to int, orElse would require making my own ISupportParameter.Modes